packages/react-storage/src/styles/storage-browser.css

```diff
@@ -96,12 +96,18 @@
   margin: 0 0 0 var(--storage-browser-gap);
 }
 
-.storage-browser__refresh {
+.storage-browser__locations-view-data-refresh,
+.storage-browser__locations-detail-view-data-refresh {
   grid-column: 5 / -1;
   justify-self: flex-end;
 }
 
-.storage-browser__refresh:has(~ .storage-browser__actions-menu) {
+.storage-browser__locations-view-data-refresh:has(
+    ~ .storage-browser__actions-menu
+  ),
+.storage-browser__locations-detail-view-data-refresh:has(
+    ~ .storage-browser__actions-menu
+  ) {
```

Comment:
What is happening here?

Reply:
Rather than having a generic classname (`.storage-browser__refresh`) we've updated them to be view specific ([here](https://github.com/aws-amplify/amplify-ui/pull/5939/files#diff-4759f6ac7ea179a389a03d228f7fef0c5d05dacc2a7b08fcc623a0d0c4e7beb9R118) and [here](https://github.com/aws-amplify/amplify-ui/pull/5939/files#diff-2a298ab664ed59a235a070ed3e4ca2e5045c4383dadd8afb5ceda79cc2c142a6R161))
 
hence updating here to reflect the same